doc/source/conf.py

```diff
@@ -118,7 +124,58 @@
 templates_path = ["_templates"]
 
 # The suffix(es) of source filenames.
-source_suffix = ".rst"
+source_suffix = {".rst": "restructuredtext", ".md": "markdown"}
 
 # The master toctree document.
 master_doc = "index"
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = "sphinx"
+
+# Execute notebooks before conversion
+nbsphinx_execute = "always"
+
+# Allow errors to help debug.
+nbsphinx_allow_errors = True
+
+# Sphinx gallery customization
+
+nbsphinx_custom_formats = {
+    ".py": ["jupytext.reads", {"fmt": ""}],
+}
```

Comment:
In the case of the magnet segmentation toolkit (or pyaedt examples rework), this lines are used to specify that we want to read (and execute, cf `nbsphinx_execute`) the examples through jupytext. I'm not sure if that it is necessary here as we don't have examples in the project.

Reply:
@SMoraisAnsys We do not have it yet. But adding examples is also in the roadmap, so we can keep them or comment them.